### PR TITLE
Delete marine.png

### PR DESCRIPTION
**Pull request by Koud-Wind**
## 🤔 What type of PR is this? (check all applicable)

- [ ] 🍕 Addition
- [ ] ⌨️ Productivity
- [ ] 🐛 Bug Fix
- [x] 🔥 Optimization
- [ ] ⚙️ Configuration
- [ ] 🌟 Quality Of Life
- [ ] ✨ Enhancement
- [ ] 📝 Documentation

## 📝 Description

Delete marine.png

## 🧐 The Rationale

optimize

## 🎯 Key Objectives

https://github.com/Cubed-Development/Modern-Warfare-Cubed/pull/128

## 🚦 Testing 

Able to operate normally

## ⏮️ Backwards Compatibility 

none

## 📚 Related Issues & Documents

https://github.com/Cubed-Development/Modern-Warfare-Cubed/pull/128

## 📖 Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 Javadoc
- [ ] 🍕 Comments
- [x] 🙅 no documentation needed

### CODE REVIEW

File: src/main/java/com/paneedah/weaponlib/EntityShellCasing.java [hunk 1]
```diff
@@ -52,7 +52,6 @@ public EntityShellCasing(PlayerWeaponInstance weaponInstance, World world, Entit
         this.weaponInstance = weaponInstance;
     }
 
-    @Override
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/weaponlib/EntityShellCasing.java:55]: Why did you remove the overide???

Comment (Koud-Wind) [reply src/main/java/com/paneedah/weaponlib/EntityShellCasing.java:55]: I always report an error every time I compile, I don't know why

Comment (Desoroxxx) [reply src/main/java/com/paneedah/weaponlib/EntityShellCasing.java:55]: What is the error?
